spec/controllers/queued_videos_controller_spec.rb

```diff
@@ -112,6 +121,130 @@
     end
   end
 
+  describe 'POST update' do
+    context 'with authentication' do
+      let(:user) { Fabricate(:user) }
+
+      before do
+        session[:user_id] = user.id
+      end
```

Comment:
1 liner `before` block would do it as well `before { session[:user_id] = user.id }`.